packages/amplify-provider-awscloudformation/src/graphql-resource-manager/amplify-graphql-resource-manager.ts

```diff
@@ -89,12 +93,14 @@ export class GraphQLResourceManager {
     const apiStack = await cfn
       .describeStackResources({ StackName: StackId, LogicalResourceId: gqlResource.providerMetadata.logicalId })
       .promise();
+
     return new GraphQLResourceManager({
       cfnClient: cfn,
       resourceMeta: { ...gqlResource, stackId: apiStack.StackResources[0].PhysicalResourceId },
       backendDir: pathManager.getBackendDirPath(),
       cloudBackendDir: pathManager.getCurrentCloudBackendDirPath(),
       rebuildAllTables,
+      disableGSILimitCheck: context.input.options[DISABLE_GSI_LIMIT_CHECK_OPTION],
```

Comment:
It'd be safer to do `context?.input?.options?.[DISABLE_GSI_LIMIT_CHECK_OPTION]`. I think there may be cases where options is undefined

Reply:
Yeah, I can null ref all the way in, no harm in doing that.